### ISSUE TITLE
Subscribing to a topic twice will cause server to crash when socket disconnects

### ISSUE THREAD

**Issue by tmcgannon** [uNetworking/uWebSockets#962]
uWebSocket.js version 16.1.0
Linux and macos
node 12.13.0

If you subscribe to the same topic more than once, the uWebSocket server process will crash when the socket disconnects.

Server.js:

```javascript
const uWS = require("uWebSockets.js");

uWS
  .App()
  .ws("/*", {
    open: (ws, req) => {
        // Subscribe twice to same topic - on socket close, server will crash (invalid pointer).
      ws.subscribe("foo");
      ws.subscribe("foo");
    },
    close: (ws, code, message) => {
      // Crash occurs after close() finishes.
    }
  })
  .listen(9001, listenSocket => {
    if (listenSocket) {
      console.log("Listening to port 9001");
    }
  });
```

Any WebSocket client will cause the server to crash. Here is a test client using the ws module:

Client.js
```javascript
const WebSocket = require("ws");

// Connect
let socket = new WebSocket("ws://localhost:9001");

// Listen for open event
socket.onopen = () => {
  // Close socket - will cause segment fault on server
  socket.close();

  // Server will crash here!
};
```

Running the server on linux will produce this error message:

```
*** Error in `node': free(): invalid pointer: 0x00007f3b4544eb58 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x70bfb)[0x7f3b45125bfb]
/lib/x86_64-linux-gnu/libc.so.6(+0x76fc6)[0x7f3b4512bfc6]
/lib/x86_64-linux-gnu/libc.so.6(+0x7780e)[0x7f3b4512c80e]
/home/project/node_modules/uWebSockets.js/uws_linux_x64_72.node(+0xa742a)[0x7f3b3e4d942a]
/home/project/node_modules/uWebSockets.js/uws_linux_x64_72.node(+0xa914e)[0x7f3b3e4db14e]
/home/project/node_modules/uWebSockets.js/uws_linux_x64_72.node(+0xbe112)[0x7f3b3e4f0112]
/home/project/node_modules/uWebSockets.js/uws_linux_x64_72.node(_ZZN3uWS16WebSocketContextILb0ELb1EE4initEvENUlPT_PciE_8__invokeI11us_socket_tEEDaS3_S4_i+0x705)[0x7f3b3e4ed1e5]
node[0x12e66a8]
node(uv_run+0x14b)[0x12d4c5b]
node(_ZN4node16NodeMainInstance3RunEv+0x337)[0xa1afa7]
node(_ZN4node5StartEiPPc+0x298)[0x9ab298]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf1)[0x7f3b450d52e1]
```

Running the server on a mac, you get:

```
node(47239,0x1112a25c0) malloc: *** error for object 0x102b18970: pointer being freed was not allocated
node(47239,0x1112a25c0) malloc: *** set a breakpoint in malloc_error_break to debug
Abort trap: 6
```

**Comment by alexhultman** [comment]
Great, this is fixed now and will release as v16.2.0

**Comment by alexhultman** [comment]
It is released now
